### DISCUSSION TITLE
Sorobanathon Submissions

### DISCUSSION THREAD

**Discussion by Secondstory17** [stellar/sorobanathon#171] [General]
Thank you for your consideration. I am submitting as of 12/15/2022, 11:39 PST.  My 7 Soroban Submissions are as follows:

Prologue:
https://nft.fimi.market/detail/nft/Stellar/GASLKN7Y67RMBILSGGXRPFL3TOYPWJKFSO6MDFOD4XL5DDMHLUGUE5HW/Pro

Chapter 1
https://nft.fimi.market/detail/nft/Stellar/GB3NDXIEMW27Z2UPM34NOWIXMMKAWL4MRM5IK3C6DN7I26D7JXCU6IY6/Door

Chapter 2
https://nft.fimi.market/detail/nft/Stellar/GDFVZ2LMFN2HEEMOU5S45CFNUTSESATNDKNNIBMOZSCHW2VH4QQVLFZ2/Lava

Chapter 3
https://nft.fimi.market/detail/nft/Stellar/GAQEYMHO7OSULQY4ZYMQKD55IUUZZA2GE777QEA3BQKJCF6F23TNXFYQ/Fire

Chapter 4
https://nft.fimi.market/detail/nft/Stellar/GBF3CINHWQLRO3PAXLLNVQHFLLWNUZXFRIY4WYR2AHSHVFE4EXSCWEU2/Wise

Chapter 5
https://nft.fimi.market/detail/nft/Stellar/GBO63YELHZT4EQUIQXFBVGGKDBWMNU5SJXS6MD2J3RKD6TWFJV4DKADG/Lake

Chapter 6
https://nft.fimi.market/detail/nft/Stellar/GB735AJHZ4P2BZ76VB4XUPFGK2JS2HNI4LSKCLDP62M6ANZ55SX3FPUR/Fish

**Comment by stellarjohn** [comment]
Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John

**Comment by stellarjohn** [comment]
Hi, we still haven't received your completed type form. Please try to resubmit. You should end up on a screen that looks like this.

![image](https://user-images.githubusercontent.com/98538609/210407229-23cf05d1-6979-4559-b9aa-261248d9c719.png)

**Comment by Secondstory17** [comment]
Hello,

I did submit it. I got the number back saying it was submitted. Should I
resubmit?

Thanks!

**Reply by stellarjohn** [reply]
Yes, once you get the KYC number, you need to go back to the typeform and enter it in along with a few other responses to complete the form.

**Comment by Secondstory17** [comment]
Okay, I did that. I guess I should redo it all, since you didn't
receive it.

**Reply by stellarjohn** [reply]
Received!

**Comment by Secondstory17** [comment]
Hello,

I wanted to double check that everything was alright with my KYC
submission. I noticed I hadn't received the XLM.

Thank you,
Ashley Felsing
